### ISSUE TITLE
Screen reader does not announce the title of the selection in drop down on Env Window

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2230]
Does not announce the titles: Overview, Packages, Intellisense

**Comment by huguesv** [comment]
That works for me, narrator says "Environment Pages", "Packages", "Combo Box", when I tab to the drop down and "Packages" is selected.

**Comment by zooba** [comment]
Yeah, I remember fixing this - guess I didn't put the issue number in the PR.
